src/Microsoft.AspNetCore.Server.IISIntegration.Tasks/build/netstandard1.0/Microsoft.AspNetCore.Server.IISIntegration.Tasks.targets

```diff
@@ -0,0 +1,35 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <_AspNetCoreIISOldMSBuild Condition=" '$(MSBuildToolsVersion)' == '14.0' Or '$(MSBuildToolsVersion)' == '12.0' ">true</_AspNetCoreIISOldMSBuild>
```

Comment:
And we don't support this because? We are built against `15`? OK with me, just want to be clear. Maybe throw a comment in.